target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -8,6 +8,10 @@ board=$(board_name)
 board_config_update
 
 case $board in
+asus,rt-n56u-b1)
+	ucidef_set_led_netdev "lan" "LAN link" "blue:lan" "br-lan" "link tx rx"
+	ucidef_set_led_netdev "wan" "WAN link" "blue:wan" "wan" "link tx rx"
```

Comment:
Isn't "link tx rx" the default?

Reply:
If I do not set the "link tx rx"  value, then the LED will not light up and it will be necessary to select them in the LED settings.

Reply:
```
ucidef_set_led_netdev() {
	local cfg="led_$1"
	local name="$2"
	local sysfs="$3"
	local dev="$4"
	local mode="${5:-link tx rx}"
```
/package/base-files/files/lib/functions/uci-defaults.sh
Minus = disable (unset)???

Reply:
Ok. I set it in defaul state